packages/react-server/src/ReactFlightReplyServer.js

```diff
@@ -247,6 +270,7 @@ function resolveModelChunk<T>(chunk: SomeChunk<T>, value: string): void {
   const resolvedChunk: ResolvedModelChunk<T> = (chunk: any);
   resolvedChunk.status = RESOLVED_MODEL;
   resolvedChunk.value = value;
+  resolvedChunk.reason = id;
```

Comment:
Wish we could give these properties virtual names. so hard to keep track of what they do in every context you use them

Reply:
I suppose we could give them helpers like `setChunkID(ResolvedModelChunk, number)` but I don't find it that big of a deal since it's only in the init case and then when you read it and in each case the variable shows what the alias means when reading the code.